swagger.json

```diff
@@ -147,12 +147,12 @@
                   "email": {
                     "type": "string",
                     "format": "email",
-                    "example": "homer.simpson@sfpp.com"
+                    "example": "homer2.simpson@sfpp.com"
                   },
                   "password": {
                     "type": "string",
                     "format": "password",
-                    "example": "donuts"
+                    "example": "donuts123@A"
```

Comment:
Similar to previous comment, I believe that the seeding script inserts the users with "donuts" password.  It would be good to retain that.